test/test_context.py

```diff
@@ -490,6 +490,150 @@ def test_cartocontext_query(self):
                 FROM cte
             ''')
 
+    @unittest.skipIf(WILL_SKIP, 'no carto credentials, skipping')
```

Comment:
Thank you for the tests

Reply:
Thanks for the comment. Adding an issue for that https://github.com/CartoDB/cartoframes/issues/600 (low priority)